moped-editor/src/utils/mapHelpers.js

```diff
@@ -142,25 +124,11 @@ export const makeCommonComponentsMapStyles = theme => ({
     padding: 0,
     position: "relative",
   },
-  geocoderContainer: {
```

Comment:
The replacement geocoder can position itself so we don't need a wrapper anymore. Some of these style overrides are moved to a file called `ProjectComponentsMap.css`